applications/applications.rst

```diff
@@ -22,6 +22,10 @@
 
             .. card:: Data Analysis
 
+            .. card:: Climatology
```

Comment:
Minor thing here, but I think we should put this under the Data Analysis category rather than keeping it as it's own header.  

I wouldn't worry about this for now though, it's more of a bigger picture organizational structure thing.